src/main/xar-resources/data/templating/templating.xml

```diff
@@ -239,61 +215,58 @@
-            therefore define a dependency on the shared-resources app (see the <link
-            xlink:href="repo">packaging documentation</link>). You can now just import the
-            module by its namespace URI, but without specifying a location:</para>
-        <programlisting language="xquery">import module namespace templates="http://exist-db.org/xquery/templates";</programlisting>
-        <para>The documentation and demo applications all read the templating module from
-            shared-resources.</para>
+            function, <code>templates:apply</code>.</para>
+        
+        <sect2 xml:id="deprecation">
+            <title>Important Note</title>
+            <para>The old version of the templating module used to be part of the <emphasis>deprecated</emphasis>
+                shared-resources package, while a <link xlink:href="https://github.com/eXist-db/templating/">new and improved version</link> 
+                is now available in its own
+                expath package. We recommend to update to the new package with the short name
+                <emphasis>templating</emphasis>, available on the eXist-db dashboard. To update, following the 3 steps below:</para>
+            <orderedlist>
+                <listitem>
+                    <para>change any dependency on shared-resources in your expath-pkg.xml to point to this package:</para>
+                    <programlisting language="xml" xlink:href="listings/listing-14.xml"/>
+                </listitem>
+                <listitem>
+                    <para>update the module URI for any imports of the templating module:</para>
+                    <programlisting language="xquery" xlink:href="listings/listing-15.txt"/>
+                </listitem>
+                <listitem>
+                    <para>New standard templating functions will go into a separate module, so you may want to add the following import in addition to the one above, which will give you access to the lib:parse-params template function (and others in the future):</para>
+                    <programlisting language="xquery" xlink:href="listings/listing-16.txt"/>
+                </listitem>
+            </orderedlist>
+        </sect2>
+        <sect2 xml:id="main-xquery">
+            <title>Calling the Templating from a Main XQuery</title>
+            <para>A complete main module which calls the
+                templating framework to process an HTML file passed in the HTTP request body could look
+                as follows:</para>
+            <programlisting language="xquery" xlink:href="listings/listing-12.txt"/>
+            <para>This module would be called from the URL rewriting controller. For example, we could
+                add a rule to <literal>controller.xq</literal> to pass any .html resource to the above
+                main query (saved to <literal>modules/view.xql</literal>):</para>
```

Comment:
Instead of `view.xql`, `view.xq` would be preferable, as it would conform to the community convention for main modules.